src/lib/peluxdevice.h

```diff
@@ -27,22 +27,69 @@
 #include "peluxdevicemanager_global.h"
 #include "peluxdevicemanagerenums.h"
 
+/*!
+ * \brief The PeluxDevice class
+ *
+ * An (almost) abstract class representing an entry in the PeluxDeviceManager model.
+ *
+ * \note Not to be instantiated directly, you can obtain instances from the model using
+ * PeluxDeviceManager::get()
+ */
 class PELUXDEVICEMANAGERSHARED_EXPORT PeluxDevice : public QObject
 {
     Q_OBJECT
 
+    /*!
+     * \copydoc PeluxDeviceManager::IdRole
```

Comment:
Because that's the name of the enum value; it's pretty standard in the Qt world to name the role names in this way (see e.g. https://code.woboq.org/qt5/qtbase/src/corelib/global/qnamespace.h.html#Qt::ItemDataRole)